### PR TITLE
Sync vs async data flows + sync/async split changes

### PR DESCRIPTION
**Pull request by tuliodesouza**

### CODE REVIEW

File: docs/FAQs/sync-vs-async.md [hunk 1]
```diff
@@ -5,31 +5,32 @@ sidebar_label: Sync vs. Async
...
--   **Accuracy**: because `async` has additional features and considers an end_user holistically, the categorisation is more accurate for businesses (this can be up to 10 percentage points more accurate and is the only way to account for inter-company transfers)
--   **Number of transactions you are able to process in a single batch**: for `sync`, you are able to send a maximum of **249 transactions in a single batch**, whereas for `async`, you are able to send a maximum of **1000 transactions in a single batch**
+
+- **Speed**: `sync` is faster with respect to processing a single batch of transactions and you do not have to rely on receiving a webhook to know when the process is complete (more information on webhooks here)
+- **Accuracy**: because `async` has additional features and considers an end_user holistically, the categorisation is more accurate for businesses (this can be up to 10 percentage points more accurate and is the only way to account for inter-company transfers)
```
Comment (dominickwok) [inline docs/FAQs/sync-vs-async.md:27]: might be worth calling out that accuracy here can refer to merchant extraction, category classifiation at a transation level, and metrics / score at a company level

File: docs/use-cases/beautiful-transactions.md [hunk 2]
```diff
@@ -16,21 +16,24 @@ Making a bank transaction feed look beautiful is one of the core applications of
...
-    1. **Note:** that in the payload, only `amount` and `description` are required. For best results, we recommend also sending `timestamp` or `date` and `end_user_id`
-    2. **Note:** If you receive transactions directly from Plaid, Ocrolus or in PDF, we allow you to just pass on the file without any manipulation. This replaces using the `/transactions` endpoint in this step. The endpoints you can use are:
-        - [Plaid - Assets Report](https://docs.herondata.io/api#tag/EndUserIntegrations/paths/~1api~1end_users~1{end_user_id_or_heron_id}~1plaid~1assets/post) 
-        - [Plaid - Transactions Report](https://docs.herondata.io/api#tag/EndUserIntegrations/paths/~1api~1end_users~1{end_user_id_or_heron_id}~1plaid~1transactions/post)
-        - [Ocrolus Report](https://docs.herondata.io/api#tag/EndUserIntegrations/paths/~1api~1end_users~1{end_user_id_or_heron_id}~1ocrolus/post)
-        - [PDF](https://docs.herondata.io/api#tag/EndUserIntegrations/paths/~1api~1end_users~1{end_user_id_or_heron_id}~1pdfs~1v1/post)
+   1. **Note:** that in the payload, only `amount` and `description` are required. For best results, we recommend also sending `timestamp` or `date` and `end_user_id`
```
Comment (dominickwok) [inline docs/use-cases/beautiful-transactions.md:31]: do we catually care about timestamp / date / end user id in snc enrichment any more? maybe not

**Review by dominickwok — APPROVED** [review]
some nits but generally looks great, thanks for pushing on this!

File: docs/FAQs/sync-vs-async.md (hunk 1 above)
Comment (tuliodesouza) [reply docs/FAQs/sync-vs-async.md]: agreed. Re-phrased the description to include your point in https://github.com/heron-data/docs/pull/58/commits/604809e239027721e5c7850afbce7a1370b44afa

File: docs/use-cases/beautiful-transactions.md (hunk 2 above)
Comment (tuliodesouza) [reply docs/use-cases/beautiful-transactions.md]: fixed in https://github.com/heron-data/docs/pull/58/commits/604809e239027721e5c7850afbce7a1370b44afa

**Review by dominickwok — APPROVED** [review]

**Pull request merged**